projects/angular/src/button/_properties.buttons.scss

```diff
@@ -4,333 +4,327 @@
       // Spacing
-      --clr-btn-vertical-margin: #{button-variables.$clr-btn-vertical-margin};
-      --clr-btn-horizontal-margin: #{button-variables.$clr-btn-horizontal-margin};
-      --clr-btn-horizontal-padding: #{button-variables.$clr-btn-horizontal-padding};
+      --clr-btn-vertical-margin: #{tokens.$cds-global-space-4};
+      --clr-btn-horizontal-margin: #{tokens.$cds-global-space-6};
+      --clr-btn-horizontal-padding: #{tokens.$cds-global-space-6};
       --clr-btn-vertical-padding: 0;
       --clr-btn-padding: var(--clr-btn-vertical-padding) var(--clr-btn-horizontal-padding);
-      --clr-btn-appearance-gap: #{variables.$clr_baselineRem_8px};
-      --clr-btn-display: inline-block;
-      --clr-btn-block-display: block;
-      --clr-btn-appearance-icon-margin: #{-1.4 * variables.$clr_baselineRem_2px};
-      --clr-btn-appearance-sm-icon-margin: -0.1rem;
 
       // Sizing
-      --clr-btn-height: #{button-variables.$clr-btn-height};
-      --clr-btn-height-sm: #{button-variables.$clr-btn-height-sm};
-
-      // Fonts
-      --clr-btn-font-weight: #{button-variables.$clr-btn-font-weight};
+      --clr-btn-height: #{tokens.$cds-global-space-11}; // this is standard height
+      --clr-btn-height-sm: #{tokens.$cds-global-space-9}; // this is btn-sm (small) height
 
       // Borders
-      --clr-btn-border-radius: #{button-variables.$clr-btn-border-radius};
-      --clr-btn-border-width: #{button-variables.$clr-btn-border-width};
-
-      // Shared colors
-      --clr-btn-transparent-bg-color: transparent;
-      --clr-btn-outline-bg-color: var(--clr-btn-transparent-bg-color);
+      --clr-btn-border-radius: #{tokens.$cds-alias-object-border-radius-100};
+      --clr-btn-border-width: #{tokens.$cds-global-space-1};
 
       // Standard button appearance - overrideable but not advertised!
-      --clr-btn-appearance-standard-line-height: #{button-variables.$clr-btn-appearance-standard-line-height};
-      --clr-btn-appearance-standard-letter-spacing: #{button-variables.$clr-btn-appearance-standard-letter-spacing};
-      --clr-btn-appearance-standard-font-size: #{button-variables.$clr-btn-appearance-standard-font-size};
-      --clr-btn-appearance-standard-font-weight: #{button-variables.$clr-btn-appearance-standard-font-weight};
-      --clr-btn-appearance-standard-height: #{button-variables.$clr-btn-appearance-standard-height};
-      --clr-btn-appearance-standard-padding: #{button-variables.$clr-btn-appearance-standard-padding};
-      --clr-btn-appearance-standard-icon-size: #{variables.$clr_baselineRem_0_5};
+      --clr-btn-appearance-standard-height: var(--clr-btn-height-sm);
+      --clr-btn-appearance-standard-padding: 0 #{tokens.$cds-global-space-6};
+      --clr-btn-appearance-standard-icon-size: #{tokens.$cds-global-space-7};
 
       // form-style button appearance styles - overrideable but not advertised!
-      --clr-btn-appearance-form-line-height: #{button-variables.$clr-btn-appearance-form-line-height};
```

Comment:
Same comment as before, we should preserve the scss variables so we can get compile-time checking since these properties can be used in so many places.

Reply:
Property in  question `clr-btn-appearance-form-line-height` was never exposed as API and should not be overridden, also all font related CSS now is handled by `generate-typography-token` mixin which is CDS token based.

Reply:
This particular property wasn't the one in question, the previous comment has details.